src/mgmt/cli_mgmt.c

```diff
@@ -549,6 +549,58 @@ int dc_mgmt_net_cfg(const char *name)
 	return rc;
 }
 
+/*
+ * Send an upcall to the agent to notify it of a clean process shutdown.
+ */
+int
```

Comment:
The return code of `dc_mgmt_disconnect()` doesn't seem to be checked anywhere, and I'm not sure what the caller could/should do if it doesn't succeed.

Reply:
Im not sure there is anything that can be done if it fails other than notify the caller that it failed and log the fact.

Reply:
Used it to print a log message in the fini call.